manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -135,3 +135,16 @@ spec:
     - record: cluster:apiserver_current_inflight_requests:sum:max_over_time:2m
       expr: |
         max_over_time(sum(apiserver_current_inflight_requests{apiserver=~"openshift-apiserver|kube-apiserver"}) by (apiserver,requestKind)[2m:])
+  - name: pre-release-lifecycle
+    rules:
+      - alert: DeprecratedAPIUsed
+        annotations:
+          message: >-
+            Deprecated API that will be removed in the next version is being used. Removing the workload that is using
+            the {{"{{$labels.group}}"}}.{{"{{$labels.version}}"}}/{{"{{$labels.resource}}"}} API might be necessary for
+            a successful upgrade to the next cluster version.
```

Comment:
Please refer to audit logs to find out the actor.

Reply:
Updated.